open-api/rest-catalog-open-api.yaml

```diff
@@ -3260,6 +3260,71 @@ components:
           additionalProperties:
             type: string
 
+    ReadRestrictions:
+      type: object
+      description: >
+          Read restrictions for a table, including projection and row filter expressions, according to the current schema.
+
+          A client MUST enforce the restrictions defined in this object when reading data
+          from the table.
+
+          These restrictions apply only to the authenticated principal, user, or account
+          associated with the client. They MUST NOT be interpreted as global policy and
+          MUST NOT be applied beyond the entity identified by the Authentication header
+          (or other applicable authentication mechanism).
+      properties:
+        required-projection:
```

Comment:
It may be "obvious" but it may be worth mentioning that the expressions are to be evaluated in the context of this table (meaning that any reference present in each `Term` instance should be interpreted as a column name from the table) so that there's no ambiguity about it (if we want to be formal about it)